src/backend/parser/parse_utilcmd.c

```diff
@@ -1314,6 +1323,52 @@ transformTableLikeClause(CreateStmtContext *cxt, TableLikeClause *table_like_cla
 		list_free(parent_extstats);
 	}
 
+	/*
+	 * Likewise, copy indexes if requested
```

Comment:
"Likewise, copy indexes if requested" is a false statement, here is only for distribution.

It would be nice not to repeat it, but it's not a big deal.